### PR TITLE
Update multihost.md

### PR DESCRIPTION
**Pull request by fincha**

### CODE REVIEW

File: docs/de/config/multihost.md [hunk 1]
```diff
@@ -6,8 +6,60 @@ editLink:    "https://github.com/ioBroker/ioBroker.docs/edit/master/docs/config/
...
-   <br><br>
-   Hilf mit bei ioBroker und erweitere diesen Artikel.  
-   Bitte beachte den [ioBroker Style Guide](community/styleguidedoc), 
-   damit die Änderungen einfacher übernommen werden können.
+## Master Konfiguration
+
+- login on master
+- run following command:
+
+> iobroker setup custom
```
Comment (GermanBluefox) [inline docs/de/config/multihost.md]: This step is only required if user has redis as State or Object database.
Please write some notice about it.

File: docs/de/config/multihost.md [hunk 2]
```diff
@@ -6,8 +6,60 @@ editLink:    "https://github.com/ioBroker/ioBroker.docs/edit/master/docs/config/
...
+Data directory (file), default[../../../iobroker-data/]: /opt/iobroker/iobroker-data/
+Host name of this machine [ioBroker-RasPi]:
+```
+> iobroker multihost enable
+``` enter pass phrase```
+> sudo service iobroker restart
+
+## Slave Konfiguration
+
+> sudo iobroker setup custom
```
Comment (GermanBluefox) [inline docs/de/config/multihost.md]: The same is here. Step is only required if redis used.

**Comment by GermanBluefox** [comment]
Thank you!

**Comment by Apollon77** [comment]
I do not understand those problems and for sure not the solutions ... do you have more details oj that "remove strict mode"?

**Comment by fincha** [comment]
@Apollon77 if you install the fresh image with iobroker and try to enable to multihost via cli you will get this error. I can't currently send you a full log. 

The error is mostly something with hex/bytes strings in strict mode - not really an issue, but you can't work without changing this.

**Comment by fincha** [comment]
@Apollon77 I was using this image: ioBroker_Image_RPi_2-3_20180401_stretch.img

**Comment by Apollon77** [comment]
So schould be 1.4 ... would be interesting if the error also occurs in js-controller 1.5

**Pull request merged**